src/eoapi/raster/setup.py

```diff
@@ -7,7 +7,7 @@
 
 inst_reqs = [
     "titiler.pgstac==0.1.0a3",
-    "jinja2>=2.11.2,<3.0.0",
+    "jinja2>=3.0,<4.0",
```

Comment:
not we could remove this once moved to the next version of titiler-pgstac (which include titiler=0.5.0)